otherlibs/dune-rpc/private/types.ml

```diff
@@ -27,7 +27,7 @@ end
 module Version = struct
   type t = int * int
 
-  let latest = 3, 18
+  let latest = 3, 19
```

Comment:
This should happen in https://github.com/ocaml/dune/pull/11478 or some dedicated PR.

Reply:
Already happened here: https://github.com/ocaml/dune/commit/d2745311bbcc76d880c01f7030f97a13b0e699c2

Reply:
Ah, very good, then it should be a fairly straightforward rebase :)

Reply:
done, thanks!